test-requirements.txt

```diff
@@ -1 +1,3 @@
+pytest
+pytest-ansible
```

Comment:
I left pytest-ansible out because of issues with python 3, no issues with leaving pytest, though.